core/frontend/src/test/utils/webpack.config.js

```diff
@@ -52,7 +52,7 @@ function createConfig(shouldInstrument) {
       new webpack.DefinePlugin({
         "process.env": Object.keys(process.env)
           .reduce((env, key) => {
-            env[key] = JSON.stringify(process.env[key]);
+            env[key] = JSON.stringify({ DEBUG: process.env[key].DEBUG });
```

Comment:
i dont think these webpack config changes look right

Reply:
Hello @aruniverse, I modified the code for security purpose based on codeQL scanning suggestions (link at the end). In short, it warns that some sensitive info such as API key can include in the public build artifact and suggests to use `DEBUG` keyword. Let me know if you still think it is not correct, I can revert them. For more info, see here
https://github.com/iTwin/itwinjs-core/security/code-scanning/44

Reply:
That's not what the article example says. the current code will try to access the "DEBUG" property of a string, which will always be `undefined`, and will also simply crash when the original key will be undefined with the message `Cannot access property "DEBUG" of 'undefined'".

What the example suggest would be `{ [key]: process.env[key]}` but be sure that the key is limited to what we need, rather than taking in all the keys of the .env object.

Reply:
Gotcha, that makes sense! I will fix these

Reply:
> be sure that the key is limited to what we need, rather than taking in all the keys of the .env object.

the issue is idk if we know ahead of time all the keys we will be using; if theres specific keys we need to avoid i guess just ignore them

Reply:
Wasnt that the reason to have IMJS prefixes, to be sure that we dont somehow use/show private stuff ?

Reply:
thats only something we do in our react-scripts fork; nothing preventing us from using any name here; but i do like the idea of enforcing that prefix here too. lets go with that @kaung001

Reply:
My bad, I thought that was the code used by react-scripts, but yeah, probably a good idea anyway so we dont unintentionnal leak stuff for our users.

Reply:
I'm not sure what the `webpack.DefinePlugin` actually do, but right now it will still bring all the keys of the `process.env`, and covert them into a huge object will have each key twice as this is used in a reducer. Is that only for reporting somewhere, or to use the `process.env` in code ? (because now to access environment variable "BOB", it will be `process.env.BOB.BOB`, instead of `process.env.BOB`.

Reply:
In any case, right now, the changes to these "process.env" should be reverted as it only makes things confusing.